hotel-admin-frontend/src/App.js

```diff
@@ -1,14 +1,43 @@
-import React from 'react';
-import { Switch, Route } from 'react-router-dom';
-import { HomePage} from './pages';
+import React,{ useEffect, createContext, useReducer, useContext } from 'react';
+import { BrowserRouter, Switch, Route, useHistory } from "react-router-dom";
+import  Home from './pages/Home';
+import  Signup  from './pages/Signup';
```

Comment:
rather importing the pages one by one please maintain a index file under /pages and export all them there and import it in a single line , see the directory hotelontouch

Reply:
Sure ! Could you tell me how to fix CI test

Reply:
click on the failed test part you will get instructions about what you did wrong